### ISSUE TITLE
fatal error: concurrent map writes

### ISSUE THREAD

**Issue by yifanes** [devopsfaith/krakend-ratelimit#4]
[krakendrate.go L61](https://github.com/devopsfaith/krakend-ratelimit/blob/master/krakendrate.go#L61
)
When I used krakend, a fatal error `fatal error: concurrent map writes` occurred when concurrency was high.

**Comment by taik0** [comment]
Hi @yifanes and welcome to the KrakenD Community!

Can you give us more information about this error?

- KrakenD version
- The full stack trace of the error
- Some way to reproduce the bug

Thank you!

**Comment by yifanes** [comment]
KrakenD version
I use https://github.com/devopsfaith/krakend-ce
version: 0.9.1
The full stack trace of the error

Some way to reproduce the bug
Just add endpoint config:
```js
      "extra_config": {
        "github.com/devopsfaith/krakend-ratelimit/juju/router": {
          "maxRate": 10000,
          "clientMaxRate": 10000,
          "strategy": "ip"

        },
```
then, I use` wrk -t4 -c1000 -d1m --script=post.lua http://127.0.0.1/api`

after start the server, this error appear

How to fix it:
I use this pr to fix it. [pr](https://github.com/devopsfaith/krakend-ratelimit/pull/5)

**Comment by wwanderley** [comment]
Same here.

**Comment by kpacha** [comment]
fixed at #5
